dist/sizzle.js

```diff
@@ -661,8 +661,19 @@ setDocument = Sizzle.setDocument = function( node ) {
 		return !el.getElementsByTagName("*").length;
 	});
 
+	//----------------------------------------------------------
+	/**
+	 * @author Skuid
+	 * In Lightning Locker, post Winter '20, the document.getElementsByClassName function is no longer the native
+	 * implementation, but it can still be used as if it were. It is safe to assume that all browsers supported by
+	 * Lightning also support this feature.
+	 * https://releasenotes.docs.salesforce.com/en-us/winter20/release-notes/rn_lwc_dom_api.htm
+	 * 
+	 * This issue is documented in CORE-2571
+	 */
 	// Support: IE<9
-	support.getElementsByClassName = rnative.test( document.getElementsByClassName );
+	support.getElementsByClassName = rnative.test( document.getElementsByClassName ) || !!window.$A;
```

Comment:
If `rnative.test( document.getElementsByClassName )` fails, then test if `window.$A` is defined. If it is, then set `support.getElementsByClassName` to true.